src/mixins/name-request-mixin.ts

```diff
@@ -19,7 +19,7 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
 
     return {
       nrNumber: nr.nrNum,
-      // TODO: Update entityType to use nr.requestTypeCd when namex supports our entity types
+      // TODO: Remove?
```

Comment:
No, we should get the entityType from the NR (and the NR is still saying it's a CORP not a BCOMP) so we will still need to fix this later.

Reply:
okie dokie